expected/wasm32-wasi-threads/predefined-macros.txt

```diff
@@ -1499,6 +1499,15 @@
 #define RRFIXEDSZ NS_RRFIXEDSZ
 #define RRQ 01
 #define RS_HIPRI 0x01
+#define RTLD_DEFAULT ((void *)0)
+#define RTLD_DI_LINKMAP 2
```

Comment:
does this make sense w/o dlinfo?

Reply:
@dicej ping

Reply:
Yeah, I don't know why this is defined unconditionally in dlfcn.h.  We could move it into the `#if defined(_GNU_SOURCE) || defined(_BSD_SOURCE)` block in that file if desired.

Reply:
I just pushed an update to address this.